### PR TITLE
Include usage for map::executeOnKeys [API-912]

### PR DESCRIPTION
**Pull request by utku-caglayan**
Modifies executeOnKey example to include executeOnKeys usage

### CODE REVIEW

File: examples/map/execute_on_key/main.go [hunk 1]
```diff
@@ -20,30 +20,41 @@ func main() {
...
+	fmt.Println(updatedTo)
+	// check the values
+	checkValue(ctx, m, "key1")
+	checkValue(ctx, m, "key2")
+	// Shutdown client
+	client.Shutdown(ctx)
+}
+
+// Retrieve and print value corresponding to "key" from map "m"
+func checkValue(ctx context.Context, m *hazelcast.Map, key string) {
```
Comment (yuce) [inline examples/map/execute_on_key/main.go:54]: No need to use context for this example, let's just use `context.TODO()` to make it simpler.

File: examples/map/execute_on_key/main.go [hunk 2]
```diff
@@ -20,30 +20,41 @@ func main() {
...
 	}
-	fmt.Println("previous value:", finalVal)
-	// check the values
-	value, err := m.Get(ctx, "key1")
+	fmt.Println("previous value:", prevVal)
+	// Observe value corresponding to "key1" changed, "key2" remains unchanged.
+	checkValue(ctx, m, "key1")
+	checkValue(ctx, m, "key2")
+	// Change both entries.
+	updatedTo, err := m.ExecuteOnKeys(ctx, &IdentifiedEntryProcessor{value: "testOnKeys"}, "key1", "key2")
```
Comment (yuce) [inline examples/map/execute_on_key/main.go:40]: How about `newValue` instead of `updatedTo`?

**Review by yuce — APPROVED** [review]
Just two trivial things.

File: examples/map/execute_on_key/main.go (hunk 2 above)
Comment (utku-caglayan) [reply examples/map/execute_on_key/main.go]: Thanks, makes sense. Fixed at https://github.com/hazelcast/hazelcast-go-client/pull/720/commits/014319c28f64f58430797e915b99c3bf348410dc

File: examples/map/execute_on_key/main.go (hunk 1 above)
Comment (utku-caglayan) [reply examples/map/execute_on_key/main.go]: Fixed at https://github.com/hazelcast/hazelcast-go-client/pull/720/commits/014319c28f64f58430797e915b99c3bf348410dc

**Review by yuce — APPROVED** [review]

File: examples/map/execute_on_key/main.go [hunk 3]
```diff
@@ -20,30 +20,41 @@ func main() {
...
+	fmt.Println(newValues)
+	// check the values
+	checkValue(m, "key1")
+	checkValue(m, "key2")
+	// Shutdown client
+	client.Shutdown(ctx)
+}
+
+// Retrieve and print value corresponding to "key" from map "m"
+func checkValue(m *hazelcast.Map, key string) {
```
Comment (nevzatseferoglu) [inline examples/map/execute_on_key/main.go:54]: Based on my reading, using `log.Fatal()` outside of the main is a bad practice. We can return `err` than check it in the main context.

File: examples/map/execute_on_key/main.go [hunk 4]
```diff
@@ -20,30 +20,41 @@ func main() {
...
+	value, err := m.Get(context.TODO(), key)
 	if err != nil {
 		log.Fatal(err)
```
Comment (nevzatseferoglu) [inline examples/map/execute_on_key/main.go:57]: Based on my reading, using `log.Fatal()` outside of the main is a bad practice. We can return `err` than check it in the main context.
Here: https://stackoverflow.com/a/33890104/10564442

**Review by nevzatseferoglu — COMMENTED** [review]
Looks great. Maybe, we can leave a tiny note about server side configuration of serialization, such as docs or guide links or something to make it more clear.

File: examples/map/execute_on_key/main.go (hunk 4 above)
Comment (utku-caglayan) [reply examples/map/execute_on_key/main.go:57]: I believe examples and tests are similar in this regard. You can check usages of it in standard library. You can see [createTestDir](https://go.dev/src/text/template/examplefiles_test.go) helper function as an example.
It is used as a helper function to examples.

**Comment by utku-caglayan** [comment]
> Looks great. Maybe, we can leave a tiny note about server side configuration of serialization, such as docs or guide links or something to make it more clear.

We provide directions for that to the user as a part of this example in "entry_processor.go".

**Review by nevzatseferoglu — APPROVED** [review]

**Pull request merged**